main/src/cgeo/geocaching/CacheDetailActivity.java

```diff
@@ -2395,14 +2395,20 @@ static void updateOfflineBox(final View view, final Geocache cache, final Resour
             final long diff = (System.currentTimeMillis() / (60 * 1000)) - (cache.getUpdated() / (60 * 1000)); // minutes
```

Comment:
performance wise it would be nice to divide only once after subtracting

Reply:
if we rename "diff" to "minutes" and introduce another statement "long days = minutes / (24 * 60);" then...